.github/workflows/main.yml

```diff
@@ -8,10 +8,10 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.19.x, 20.11.x, 22.0.x]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
We need to refine the versions
https://angular.dev/reference/versions
For angular 19.2: `^18.19.1 || ^20.11.1 || ^22.0.0`